dbms/src/Common/TiFlashMetrics.h

```diff
@@ -95,21 +95,20 @@ namespace DB
         F(type_alter_column_tp, {"type", "alter_column_type"}), F(type_rename_column, {"type", "rename_column"}),                         \
         F(type_exchange_partition, {"type", "exchange_partition"}))                                                                       \
     M(tiflash_schema_apply_duration_seconds, "Bucketed histogram of ddl apply duration", Histogram,                                       \
-        F(type_ddl_apply_duration, {{"req", "ddl_apply_duration"}}, ExpBuckets{0.0005, 2, 20}))                                           \
+        F(type_ddl_apply_duration, {{"req", "ddl_apply_duration"}}, ExpBuckets{0.001, 2, 20}))                                            \
     M(tiflash_tmt_merge_count, "Total number of TMT engine merge", Counter)                                                               \
     M(tiflash_tmt_merge_duration_seconds, "Bucketed histogram of TMT engine merge duration", Histogram,                                   \
-        F(type_tmt_merge_duration, {{"type", "tmt_merge_duration"}}, ExpBuckets{0.0005, 2, 20}))                                          \
+        F(type_tmt_merge_duration, {{"type", "tmt_merge_duration"}}, ExpBuckets{0.001, 2, 20}))                                           \
     M(tiflash_tmt_write_parts_count, "Total number of TMT engine write parts", Counter)                                                   \
     M(tiflash_tmt_write_parts_duration_seconds, "Bucketed histogram of TMT engine write parts duration", Histogram,                       \
-        F(type_tmt_write_duration, {{"type", "tmt_write_parts_duration"}}, ExpBuckets{0.0005, 2, 20}))                                    \
-    M(tiflash_tmt_read_parts_count, "Total number of TMT engine read parts", Gauge)                                                       \
+        F(type_tmt_write_duration, {{"type", "tmt_write_parts_duration"}}, ExpBuckets{0.001, 2, 20}))                                     \
```

Comment:
these metrics can also be removed

* tiflash_tmt_merge_count
* tiflash_tmt_merge_duration_seconds
* tiflash_tmt_write_parts_count
* tiflash_tmt_write_parts_duration_seconds
* tiflash_tmt_read_parts_count